rll/setup-automatic-upgrade.sh

```diff
@@ -44,28 +50,51 @@ else
     rsc_command="schedule_recipe /api/right_net/scheduler/schedule_recipe recipe=rll::upgrade"
   fi
 
+  # Random hour 0-23
+  scheduled_hour=$(( $RANDOM % 24 ))
+
+  # Random minute 0-59
+  scheduled_minute=$(( $RANDOM % 60 ))
+
+  if [[ $(cat /etc/os-release) =~ CoreOS ]]; then
```

Comment:
Because the condition is wrapped around `[[` and `]]`, it will not error out, however the error will be sent to STDERR. Perhaps redirecting STERR to /dev/null as not to show the error to customers:
`if [[ $(cat /etc/os-release 2> /dev/null) =~ CoreOS ]]; then`